packages/web3-eth/test/integration/subscription_heads.test.ts

```diff
@@ -15,38 +15,31 @@ You should have received a copy of the GNU Lesser General Public License
 along with web3.js.  If not, see <http://www.gnu.org/licenses/>.
 */
 import { BlockHeaderOutput } from 'web3-types';
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { Web3 } from 'web3';
 import { Web3Eth, NewHeadsSubscription } from '../../src';
-import { Resolve, sendFewTxesWithoutReceipt } from './helper';
+import { Resolve } from './helper';
 import {
 	closeOpenConnection,
-	createTempAccount,
 	describeIf,
 	getSystemTestProvider,
 	isSocket,
+	sendFewSampleTxs,
 	waitForOpenConnection,
 } from '../fixtures/system_test_utils';
 
-const checkTxCount = 3;
-
+const checkTxCount = 2;
 describeIf(isSocket)('subscription', () => {
 	let clientUrl: string;
-	let tempAcc2: { address: string; privateKey: string };
-
-	beforeEach(async () => {
-		tempAcc2 = await createTempAccount();
-	});
-	beforeAll(() => {
+	let web3: Web3;
+	beforeAll(async () => {
 		clientUrl = getSystemTestProvider();
 	});
```

Comment:
This `beforeAll` doesn't need to be here, can assign `clientUrl` upon initialization

Reply:
done